modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -239,9 +239,9 @@ public void testNoInteractionsAfterStop() throws Exception {
         CompletableFuture<Void> readyFuture = manager.catalogReadyFuture(1);
         assertFalse(readyFuture.isDone());
 
-        manager.stop();
+        manager.stopAsync();
```

Comment:
Here and in other places, let's add verification that corresponding startAsync/stopAsync futures were completed successfully. I'm talking about `assertThat(manager.stopAsync(), willCompleteSuccessfully());` In some places we may combine components start/stop futures and assert that combined one was completed successfully - not mandatory action though.